#9 login form patch

radumg opened the pull request:
fixes #9 

cc @didimitrie

mswaidan commented:
See comments on the original issue, but we also want to keep URL at 0.0.0.0. This allows dev servers to be accessed from addresses not on the LAN. E.g. I run my dev server on Digital Ocean rather than on localhost.

didimitrie commented:
plus `package-lock.json` is a godsend, it needs to stay!

didimitrie commented on src/components/LoginForm.vue:
```diff
@@ -27,7 +27,7 @@
                   <label>Password</label>
-                  <md-input v-model="password" type="password" v-validate name="password" data-vv-rules="min:8|required" data-vv-name="password" autocomplete="password"></md-input>
+                  <md-input v-model="password" type="password" v-validate name="password" data-vv-name="password" autocomplete="password"></md-input>
                   <span v-show="errors.has('password')" class="md-error">{{ errors.first('password') }}</span>
```
also all the validate directives plus span below need to removed if we don't validate the passwd lenght

radumg replied:
👍

radumg commented:
ok, reverted the `localhost` change, but i think a more elegant solution is needed, the experience whilst developing `SpeckleAdmin` locally with the server remotely is currently a bit broken, `0.0.0.0` is launched.
Also see #11 , hot reloading wasn't working.

also reverted `package-lock.json` - if you're happy to have this constantly updated when we run `npm install` 👍

radumg commented:
btw, thanks for the uber-quick replies @didimitrie & @mswaidan 🍻

radumg commented:
Should be ok to merge now ?

radumg commented:
server-side fixed by @didimitrie [here](https://github.com/speckleworks/SpeckleServer/commit/df10599f5b9ef28eb1232ea18ae96862ccec7bc5)
